packages/toolpad-app/src/toolpad/ToolpadShell/Header/index.tsx

```diff
@@ -24,80 +24,83 @@ function Header({ actions, status, enableUserFeedback = true }: HeaderProps) {
+            sx={{
+              flex: 1,
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'start',
+            }}
+          >
+            <Tooltip title="Home">
+              <Link
+                color="inherit"
+                aria-label="Home"
+                href="/"
+                underline="none"
+                sx={{ display: 'flex', flexDirection: 'row', alignItems: 'center', gap: 1 }}
               >
-                MUI Toolpad
-              </Box>
-            </Link>
-          </Tooltip>
-          <Chip sx={{ ml: 1 }} label="Alpha" size="small" color="grey" />
-        </Box>
-        <Box
-          sx={{
-            flex: 1,
-            display: 'flex',
-            flexDirection: 'row',
-            alignItems: 'center',
-            justifyContent: 'center',
-          }}
-        >
-          {actions}
-        </Box>
-        <Box
-          sx={{
-            flex: 1,
-            display: 'flex',
-            flexDirection: 'row',
-            alignItems: 'center',
-            justifyContent: 'end',
-            gap: 2,
-          }}
-        >
-          {status}
-          <ThemeModeMenu mode={themeMode} onChange={handleThemeModeChange} />
-          {enableUserFeedback ? <UserFeedback /> : null}
-        </Box>
-      </Toolbar>
-    </AppBar>
+                <Image
+                  src={theme.palette.mode === 'dark' ? productIconDark : productIconLight}
+                  alt="Toolpad product icon"
+                  width={25}
+                  height={25}
+                />
+                <Box
+                  data-testid="brand"
+                  sx={{
+                    color: 'primary.main',
+                    lineHeight: '21px',
+                    fontSize: '16px',
+                    fontWeight: 700,
+                    letterSpacing: 0,
+                    fontFamily: theme.typography.fontFamily,
+                  }}
+                >
+                  MUI Toolpad
+                </Box>
+              </Link>
+            </Tooltip>
+            <Chip sx={{ ml: 1 }} label="Alpha" size="small" color="grey" />
+          </Box>
+          <Box
+            sx={{
+              flex: 1,
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'center',
+            }}
+          >
+            {actions}
+          </Box>
+          <Box
+            sx={{
+              flex: 1,
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              justifyContent: 'end',
+              gap: 2,
+            }}
+          >
+            {status}
+            <ThemeModeMenu mode={themeMode} onChange={handleThemeModeChange} />
+            {enableUserFeedback ? <UserFeedback /> : null}
+          </Box>
+        </Toolbar>
+      </AppBar>
+      <Toolbar />
```

Comment:
Adds spacing because `AppBar` position was changed to `fixed` so that it's not contrained by the new sidebar. As per recommended in MUI Drawer docs https://mui.com/material-ui/react-drawer/ ("Clipped under the app bar")